.github/workflows/lint_test_build.yml

```diff
@@ -102,6 +102,27 @@ jobs:
         run: |
           npx @stoplight/spectral-cli lint ./packages/token-introspection/openapi/*.yaml
 
+  integration-test:
+    runs-on: ubuntu-22.04
+    needs: checkout
+    timeout-minutes: 5
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup environment
+        uses: ./.github/workflows/rafiki/env-setup
+
+      - name: Setup hosts
+        run: |
+          echo "127.0.0.1 host.docker.internal" | sudo tee -a /etc/hosts
```

Comment:
We're eventually going to use sudo and hostile to do this in the `run-tests` script but I think this will end up being permanent for the reason outlined in the end of the body in this issue: https://github.com/interledger/rafiki/issues/2555

Reply:
I'm not sure I'm following. Why do we need it twice? Because that way, the script itself doesn't prompt?

Reply:
Good questions - short answer is that I wonder if hostile will work fine in CI after all, in which case we can get rid of this step. I will test that.

The thinking was hostile will not work in github actions due to sudo prompt (needs confirming). Thus the extra step. And that appending to `/etc/hosts` and otherwise manually performing a "append if not exist" was better managed by something like hostile, thus not using  `echo "127.0.0.1 host.docker.internal" | sudo tee -a /etc/hosts`. And to answer Max's question, that does actually prompt from the command line - that's what makes me think my original assumptions about github actions and sudo might just be bad.

Also questioning if need hostile at this point. We are already checking for the existence of some line before setting a new one, so we shouldn't be adding duplicates. Beyond that and being cross-platform, which I don't think matters because we're already assuming a unix-like environment (including WSL) with this shell script and elswhere, it basically just does this:
- erroring if you try to overwrite local or localhost or the cli args are missing https://github.com/feross/hostile/blob/master/bin/cmd.js#L62-L70
- not setting if it exists (we're already checking that) https://github.com/feross/hostile/blob/master/index.js#L93
- adding a newline such that it preserves any existing whitespace at the end of the file.  https://github.com/feross/hostile/blob/master/index.js#L94-L101

So I'm not really seeing a value-add over checking if a line already exists and if not adding it.

Reply:
I guess I forgot where I was... editing `/etc/hosts` via hostile in `run-tests` is not in this branch and adding host step is required regardless here.

I will test using hostile in CI on that other branch when this is merged.